mathesar/state/django.py

```diff
@@ -147,8 +146,7 @@ def _invalidate_columns_with_incorrect_display_options(tables):
             )
             if not serializer.is_valid(False):
                 columns_with_invalid_display_option.append(column.id)
-    if len(columns_with_invalid_display_option) > 0:
-        models.Column.current_objects.filter(id__in=columns_with_invalid_display_option).update(display_options=None)
+    models.Column.current_objects.filter(id__in=columns_with_invalid_display_option).update(display_options=None)
```

Comment:
You're right that the length check here is unnecessary, but it was beneficial for readability, since it made it easier to tell when database queries did and didn't happen. I'd like this change removed.